ndt_scan_matcher_node:
  ros__parameters:
    ndt_scan_matcher:
      topic_name:
        cloud: "/camera/depth_registered/points"
        # estimate_pose: "grid_point_observer/estimate_pose"
        estimate_pose: "ndt_scan_matcher/laser_pose"
        estimate_twist: "grid_point_observer/estimate_twist"
      tf_frame:
        map_frame: "map"
        odom_frame: "odom"
        robot_frame: "base_link"
      broadcast_period: 0.001
      odom_tf_broadcast: true
      # 点群パラメータ
      min_point_cloud_size: 100
      filter:
        voxelgrid_size: 0.06
      ndt:
        trans_epsilon: 0.00000001 #0.00000001
        step_size: 0.06
        resolution: 0.06
        max_iterations: 1000.0
        min_score_limit: 0.001
      #カルマンフィルタ設定
      kalman_filter:
        Q: [0.2, 0.2, 0.2, 0.001, 0.001, 0.001] #システムノイズの共分散行列 (n*n) def [0.2, 0.2, 0.2, 0.2, 0.2, 0.2]
        R: [0.7, 0.7, 0.7, 0.2, 0.2, 0.2] #観測ノイズの共分散行列 (p*p) def [0.7, 0.7, 0.7, 0.7, 0.7, 0.7]
